modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -885,6 +885,27 @@ RETURN 'val' as one, 'val' as two
 ----
 ====
 
+.Using id() function
+====
+Query::
++
+[source,cypher]
+----
+MATCH (a)
+RETURN id(a)
+----
+Description of the returned code::
```

Comment:
Can we update the description or it's an old one?

Reply:
It is an old one